ee/api/test/test_event_definition.py

```diff
@@ -0,0 +1,29 @@
+from typing import cast
+
+from django.utils import timezone
+from rest_framework import status
+
+from ee.models.event_definition import EnterpriseEventDefinition
+from ee.models.license import License, LicenseManager
+from posthog.test.base import APIBaseTest
+
+
+class TestEventDefinitionEnterpriseAPI(APIBaseTest):
+    def test_event_description(self):
```

Comment:
I think we need to add a bit more test cases here. Some come to mind right away: making sure you can't edit stuff that you don't have permission to, updating tags, setting a blank description, making sure `name`, `updated_at`, .... can't be updated directly, making sure `updated_at` & `updated_by` are properly updated